src/pages/Register.tsx

```diff
@@ -0,0 +1,160 @@
+import { createUserWithEmailAndPassword } from 'firebase/auth';
+import React, { useState } from 'react';
+import { auth, db } from '../firebase/index'
+import { setDoc, doc } from 'firebase/firestore';
+import { useNavigate } from 'react-router-dom';
+
+export function Register() {
```

Comment:
para el caso de los componentes que son páginas, los nombres terminarían en Page, ejemplo:
```suggestion
export function RegisterPage() {
```